build-tools/src/main/resources/plugin-descriptor.properties

```diff
@@ -1,5 +1,5 @@
 # Elasticsearch plugin descriptor file
-# This file must exist as 'plugin-descriptor.properties' inside a plugin.
+# This file must exist as 'plugin-descriptor.properties' or 'stable-plugin-descriptor.properties inside a plugin.
```

Comment:
Somewhat unrelated - is the name of this file still ok?

Reply:
good point, perhaps plugin-descriptor-template.properties? or something along these lines (ideas?)
this is a file that is used by both stable and old-style gradle plugins to generate plugin-descriptor.properties (so this file but with values populated)

Reply:
Yeah, 'template' is a good idea, that would work